docs/development/deploying-locally/index.md

```diff
@@ -1,10 +1,11 @@
 # Deploying locally
 
-REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+REANA can be easily deployed locally on your laptop using _Kubernetes in Docker_ ([kind](https://kind.sigs.k8s.io/)).
+Useful for REANA platform development.
 
 ## Instructions
 
-**1.** [Fork the REANA main repository](https://github.com/reanahub/reana/fork) and clone it to install `reana-dev` helper script:
+**1.** [Fork the REANA main repository](https://github.com/reanahub/reana/fork) and clone it to install `reana-dev` CLI:
```

Comment:
I would leave `helper script` here... It better explains that this is something that only "knowledgeable users" should use... Not meant to be a CLI for everybody... That kind of impression.

Reply:
My original impression about the `helper script` expression was that it reflected that it is something _unofficial_ and _temporal_, up to some degree. I felt that calling it `CLI` is a bold adjective reflecting _non-temporality_.

Also, most of the user base, not only _knowledgeable users_ are going to use the `reana-dev`. For instance, the command `eval $(reana-dev client-setup-environment)` is quite useful, even for _rookie_ users.

That being said, it is just a detail. I will revert my change if there is no further discussion on this.